assets/models/system/Mixtral-8x7B-v0-1/spec.yaml

```diff
@@ -20,8 +23,17 @@ tags:
       Standard_ND96asr_v4,
     ]
   inference_supported_envs:
-  - vllm
+    - vllm
+  finetune_compute_allow_list:
+    - Standard_ND40rs_v2
+    - Standard_ND96amsr_A100_v4
```

Comment:
Add other NDA100 SKU as well, check other model cards for it

Reply:
I believe it is Standard_ND96asr_v4, added.